docs/src/examples/components/Form/Types/FormExampleDropdown.shorthand.tsx

```diff
@@ -0,0 +1,31 @@
+import * as React from 'react'
+import { Form, Dropdown, Button } from '@stardust-ui/react'
+
+const labelId = 'choose-friend-label'
+const fields = [
+  {
+    label: { content: `Your best friend's name is:`, id: labelId },
```

Comment:
I am surprised with generated markup, is it valid? I expect:
```html
<label for="favoritefood">Favorite food</label> <!-- <<== `for` is used -->
<select name="favoritefood" id="favoritefood">
  <option>Cheese</option>
</select>
```

We have:

```html
<label class="ui-text" dir="auto" id="choose-friend-label">Your best friend&#x27;s name is:</label>
<div class="ui-dropdown ui-box" required="" name="chooseFriend" />
```

And if we will pass `id` on root:

```html
 <label class="ui-text" dir="auto" for="choose-friend-label" id="choose-friend-label">
```

Reply:
About `label` & `htmlFor`: https://codesandbox.io/s/dark-microservice-47wft